deployment/prepare.yaml

```diff
@@ -69,7 +69,6 @@
         creates: "{{service_folder}}/{{tomcat}}"
         src: "http://archive.apache.org/dist/tomcat/tomcat-8/v{{tomcat_version}}/bin/{{tomcat}}.tar.gz"
         dest: "{{service_folder}}"
-        copy: no
```

Comment:
why are you deleting this?
Doesn't this mean that the download will happen every time?

Reply:
No it doesn't. On my verison of ansible, it says that "copy" and "remote_src" are mutually exclusive, and the script fails unless that is removed.